### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pachakutiq199999**
Added DNSExit in Email section.
Added mogenius in web hosting section.
Updated Yandex.Connect in Email section.

### CODE REVIEW

**Comment by pachakutiq199999** [comment]
698C55D6-77AA-4693-B324-D823BF808960

File: README.md [hunk 1]
```diff
@@ -823,7 +824,7 @@ Table of Contents
   * [Validator.Pizza](https://www.validator.pizza/) — Free API to detect disposable emails
   * [Verifalia](https://verifalia.com/email-verification-api) — Real-time email verification API with mailbox confirmation and disposable email address detector; 25 free email verifications/day.
   * [verimail.io](https://verimail.io/) — Bulk and API email verification service. 100 free verifications/month
-  * [Yandex.Connect](https://connect.yandex.com/pdd/) — Free email and DNS hosting for up to 1,000 users
+  * [Yandex.Connect](https://connect.yandex.com/pdd/) — Free email and DNS hosting for up to 1,000 users *(currently not available for new users to register)*
```
Comment (ripienaar) [inline README.md:827]: so we can just remove it i think

Comment (pachakutiq199999) [reply README.md:827]: I guess so.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**